### ISSUE TITLE
[Feature Request] Higher Contrast Buttons for Better Accessibility

### ISSUE THREAD

**Issue by OverthinkingTech** [jmshrv/finamp#902]
Currently, the play/pause, skip track, shuffle, etc. buttons in the player change icon color based on the album artwork. This creates many scenarios in which those buttons are very hard to see. Ideally I would like to see and option for high contrast buttons that sets the icon color to white -while in dark mode- and black -while in light mode-

**Comment by Chaphasilor** [comment]
I've been thinking about just a high-contrast mode for a while. Would some sort of higher saturation mode also work? Or applying a slight tint to the black/white?

**Comment by Chaphasilor** [comment]
Added in 9fce655b95b49a4b028a263f2c5c590df32ba081
